boards.txt

```diff
@@ -38,6 +38,8 @@ esp32_family.name=ESP32 Family Device
 esp32_family.hide=true
 esp32_family.vid.0=0x303a
 esp32_family.pid.0=0x1001
+esp32_family.upload_port.0.vid=0x303a
+esp32_family.upload_port.0.pid=0x1001
```

Comment:
@ladyada in order for discovery to work properly with IDE2 and CLI, it is necessary to add the `.upload_port` lines. Maybe someone there can update your board definitions?

cc: @UnexpectedMaker

Reply:
thanks
@hathach can you add to your ToDo list :)
@me-no-dev is there a date you need this by?

Reply:
@ladyada we plan on pushing bugfix releases fairly often, so it's not a hurry.

Reply:
@ladyada sure, I will follow this up